pkg/master/master.go

```
@@ -586,11 +590,20 @@ func (m *Master) initV1ResourcesStorage(c *Config) {
 }
 
 // NewBootstrapController returns a controller for watching the core capabilities of the master.
-func (m *Master) NewBootstrapController() *Controller {
+func (m *Master) NewBootstrapController(c *Config) *Controller {
+	var masterLeases leases.Leases
+	if c.EnableLeases {
+		leaseStorage, err := c.StorageFactory.New(api.Resource("apiServerIPInfo"))
+		if err != nil {
+			glog.Fatalf(err.Error())
```

Should really return an error, not glog fatal.